### DISCUSSION TITLE
PPGW新增节点无法测速

### DISCUSSION THREAD

**Discussion by fannagi** [kkkgo/PaoPaoGateWay#108] [问和答]
PPGW搭配PPDNS使用了有快半年，没出现过什么毛病。不过最近梯子抽风得厉害，另外加了新买机场的节点后，在PPGW上新机场使用的节点均无法测速，原来旧节点能使用的可正常测速。新机场节点在手机CFW中是可以正常使用的。

https://github.com/kkkgo/PaoPaoGateWay/issues/20

目前已参考该例子，在 dns_burn 和 ex_dns 中添加 PPDNS 地址

dns_ip=192.168.50.6
dns_port=5304

dns_burn=yes
ex_dns="192.168.50.6:5304,223.5.5.5:53,114.114.114.114:53"

PPDNS使用的是2、3月的版本，已在 force_nocn_list.txt 文件中添加相应域名。在电脑Ping 之后的结果与在 https://ping.chinaz.com/  中测试的不算完全一致，不知道是不是新机场用了 Cloudflare CDN ，这边每隔段时间Ping 出来的结果都会变动，网站上也Ping 出来几十个 IP 。

现在想问下标题中的问题得怎么解决？

**Comment by kkkgo** [comment]
dns_burn只有在fast_node开启的时候生效

**Reply by fannagi** [reply]
目前 fast_node=check 我改成 yes 再试试。主要是现在用的 Fallback ，不行会自动切换，感觉 check 应该也是可以生效的？

**Reply by kkkgo** [reply]
dns_burn功能会对订阅配置硬编码DNS解析结果，也就是配置需要完全由ppgw管理，所以依赖fast_node=yes。

**Reply by fannagi** [reply]
刚刚改为 fast_node=yes 也是不行，和原来一样的效果，大佬再帮忙看看哪里有可能出问题？

**Reply by kkkgo** [reply]
你重启一下再看看吧，生效的话你面板应该看到节点被裂变成了多倍才对。

**Reply by kkkgo** [reply]
另外dns_burn仅适用于节点本身在配置文件里面的情况，不适用于provider等引用。

**Reply by fannagi** [reply]
这边用电信是能Ping通的，用联通就不行，大概率是运营商问题了，有点无语。感谢大佬帮忙分析。

**Reply by fannagi** [reply]
最后找到原因了，原来是内核问题。一直用的原版内核，现在用 mihomo 最新版本能用了。我有点好奇为啥原版不支持 vmess 。

**Reply by kkkgo** [reply]
原版就是被删库的那个clash原版，支持的协议是没有很多。对于大多数中转代理来说实际上都是连的中转机器，协议无所谓，用的都是很简单ss协议，足够使用，体积比mihomo小很多。

https://github.com/kkkgo/PaoPaoGateWay/discussions/33

**Reply by fannagi** [reply]
因为之前一直在用 Trojan 都正常，没想到先支持 Trojan 也没支持到 vmess ，也就没往那方面想
